src/components/Overview/Overview.react.js

```diff
@@ -149,8 +148,7 @@ class Overview extends Component {
       }
     const actions = <RaisedButton
       label="Cancel"
-      backgroundColor={
-        UserPreferencesStore.getTheme()==='light' ? '#4285f4' : '#19314B'}
+      backgroundColor='#607D8B'
```

Comment:
@fragm3  #4285f4 this one is the new colour.#607D8B is older one .please swap that.